<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModBusSettings</class>
 <widget class="QDialog" name="ModBusSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>aqmodbus.png</normaloff>aqmodbus.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(133, 192, 24);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabCon">
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <widget class="QLabel" name="lbCoil">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Coil (8-bit words)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tCoilNum">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbAnaIn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Analog Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input (8-bit words)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tInAddr">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPaddr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="tCoilAddr">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="tAnaNum">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>16</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="tInNum">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblIPport">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPport">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>502</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="tRegNum">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>32</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbStart">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbReg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tAnaAddr">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblSlaveID">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Slave</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tRegAddr">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbNumber">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIIPaddr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cancel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="slaveID">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabUI">
    <attribute name="title">
     <string>Interface</string>
    </attribute>
    <widget class="QPushButton" name="btnBGColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>291</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Under Construction</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAbout">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QTextBrowser" name="tbAbout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;AQModBus v0.1.90&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Source and forum located at http://modbus.makrit.net&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Copyright © 2011 Mark Gullings http://www.makrit.net&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Which is nearly a complete ripoff of QModBus © 2009 - Tobias Doerffel - http://qmodbus.sourceforge.net&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;It incorporates libmodbus © 2001-2008 - Stéphane Raimbault - http://www.libmodbus.org&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;And incorporates QLedMatrix © 2009 Pierre-Etienne Messier - http://pemessier.hexpresso.org&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;You should have received a copy of the GNU General Public License along with this program (see COPYING); if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Connect</sender>
   <signal>clicked()</signal>
   <receiver>ModBusSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ModBusSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
